CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ x.x.x Release notes (yyyy-MM-dd)
 
 ### Fixed
 * <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-js/issues/????), since v?.?.?)
-* None.
+* When creating a class that extends Realm.Object and pushing the instantiated object to a list, a segmentation fault would occur.  This has been fixed by a null check and afterwards creating the object. 
```

Comment:
Do it also solve https://github.com/realm/realm-js/issues/2469? If so, we should add a link to it.

Reply:
Unfortunately it does not.